test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -259,10 +259,10 @@ UCS_TEST_P(test_uct_sockaddr, many_clients_to_one_server)
 
 UCS_TEST_P(test_uct_sockaddr, many_conns_on_client)
 {
-    int i, num_conns_on_client = 100;
+    int num_conns_on_client = ucs_max(2, 100 / ucs::test_time_multiplier());
```

Comment:
reducing the number of tested clients under valgrind to 5 is basically not testing any load under valgrind.
maybe test half (50) with valgrind?

Reply:
valgrind testing significantly increase timings to mark memory for both client and server, I don't think it's a real case for high load testing.